packages/connect/src/api/ethereum/__fixtures__/ethereumSignTx.ts

```diff
@@ -1,12 +1,21 @@
-export const serializeEthereumTx = [
+import { LegacyTxData } from '@ethereumjs/tx';
+
+interface EthereumTxFixture {
+    description: string;
+    chainId: number;
+    tx: LegacyTxData; // + TODO: FeeMarketEIP1559TxData
```

Comment:
woudl be nice to add a fixture to cover this https://github.com/trezor/trezor-suite/actions/runs/11714141343/job/32643617804#step:16:1345